client/modules/core/main.js

```diff
@@ -243,6 +243,14 @@ export default {
     return settings.settings || {};
   },
 
+  getShopCurrency() {
+    const shop = Shops.findOne({
+      _id: this.shopId
+    }) || {};
+
+    return shop.currency;
```

Comment:
How do we handle this usually then? I just reference the function right above it which does the same thing:

```
getShopSettings() {
    const settings = Packages.findOne({
      name: "core",
      shopId: this.shopId
    }) || {};
    return settings.settings || {};
  },
```